kb/2024-01-30/the_potential_risk_with_fstrim.md

```diff
@@ -60,4 +96,18 @@ You can determine if fstrim is enabled in VMs that use affected Longhorn volumes
     AccuracySec=1h
     Persistent=true
     RandomizedDelaySec=6000
-    ```
\ No newline at end of file
+    ```
+
+### Windows
+
+Disable the TRIM operation on Windows will mitigate the described risks, you can refer the following command:
+
+- To disable TRIM operation on ReFS v2:
+```
```

Comment:
indent the code blocks for consistency. Check the preview: https://deploy-preview-59--harvester-home-preview.netlify.app/kb/the_potential_risk_with_fstrim